package-lock.json

```diff
@@ -1,23 +1,24 @@
 {
   "name": "@eclipse-velocitas/velocitas-project-generator",
   "version": "3.0.1",
-  "lockfileVersion": 2,
+  "lockfileVersion": 3,
   "requires": true,
   "packages": {
     "": {
       "name": "@eclipse-velocitas/velocitas-project-generator",
       "version": "3.0.1",
       "license": "Apache-2.0",
       "dependencies": {
-        "axios": "^0.27.2",
+        "axios": "^1.6.7",
```

Comment:
Axios was deliberately kept on this low version. @dennismeister93  would need to perform additional testing before pulling up the version to this new one.

Reply:
Without bulking a version, we will keep 2 moderate vulnerability.
I will create follow up story and will link those 2 risks as dependency.